apps/nextjs/src/components/post.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface PostProps {
+  data: {
+    id: number;
+    title: string;
+    contents: string;
+    createdAt: Date;
+  };
+  comments: {
+    id: number;
+    postId: number;
+    text: string;
+    createdAt: Date;
+  }[];
+}
+
+const Post: React.FC<PostProps> = ({ data, comments }) => {
```

Comment:
just a question, but can you destructure the data object here instead of in the next line?